continous_integeration/environment-3.7.yaml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.7
-  - dask[dataframe,distributed]
+  - dask
```

Comment:
This is just a cleanup change. The `[dataframe,distributed]` extra is a `pip`-only feature, so is I think currently being ignored anyways